Terminal.Gui/Core/Window.cs

```diff
@@ -97,9 +97,9 @@ public override void OnCanFocusChanged ()
 				base.OnCanFocusChanged ();
 			}
 
-			public override bool MouseEvent (MouseEvent mouseEvent)
+			public override bool OnMouseEvent (MouseEvent mouseEvent)
```

Comment:
Isn't renaming a public member a breaking change?

Reply:
OnMouseEvent already exists and it will also call the MouseEvent and I didn't needed to change any unit test because of this change.

Reply:
This goes unnoticed for users and does not influence their functioning at all, because it's implemented on the private `ContentView` sub-class of the `Window` class.